src/Dotnet.Script.Tests/TestFixtures/Issue435/Issue435.csx

```diff
@@ -0,0 +1,6 @@
+﻿#r "nuget: Microsoft.Extensions.Configuration"
```

Comment:
this has no version by design, so that we fetch the latest. If there is a new major/minor released, this test should break